package.json

```diff
@@ -7,6 +7,11 @@
     "type": "git",
     "url": "git+https://github.com/frandiox/vite-ssr.git"
   },
+  "scripts": {
+    "dev": "tsc -w -p tsconfig.json",
+    "build": "rm -rf dist && tsc -p tsconfig.json && cat ./package.json | grep -v '\"private\":' > dist/package.json"
+  },
+  "private": true,
```

Comment:
followed the pattern you suggested in vitedge here.

Reply:
I think we can use the same, yes 👍